src/heroes/sombra/translocator_cooldown_reset.opy

```diff
@@ -0,0 +1,64 @@
+#Global variables
+
+globalvar STEALTH_REFUND_FRACTION 0
+globalvar NEGATIVE_STATUS_PREVENTS_REFUND 1
+globalvar MINIMUM_TRANSLOCATOR_COOLDOWN 2
+
+
+#Player variables
+
+playervar stealth_timer 0
+playervar stealth_max_Cooldown 1
+playervar ult_charge_Percent_cache 2
+
+
+rule "Initial Global":
```

Comment:
Make rule name format match the rest of the repository `[file_path/file_name.opy]: rule description`

Reply:
it follows the format now, but please review if the descriptions are useful